src/main/cpp/NetworkTableUtil.cpp

```diff
@@ -0,0 +1,16 @@
+// Copyright (c) 2021 FRC Team 3512. All Rights Reserved.
+
+#include "NetworkTableUtil.hpp"
+
+namespace frc3512::NetworkTableUtil {
+
+nt::NetworkTableEntry MakeEntry(nt::NetworkTableInstance& instance,
+                                const wpi::Twine& name,
+                                std::initializer_list<double> defaultValue) {
+    nt::NetworkTableEntry entry = instance.GetEntry(name);
```

Comment:
Since we only ever use the default NT instance, we could just grab it in here and not take it as an argument. Then the m_inst variables in all the subsystems can be removed.

Reply:
good idea